checkNodeVersion.js

```diff
@@ -0,0 +1,21 @@
+const result = process.versions
+if (result && result.node) {
+  if (parseInt(result.node) >= 14) {
```

Comment:
we are expecting at least the 15th version of Node in `@sasjs/adapter`, so it makes sense to stay consistent